docs/components/modeler/bpmn/bpmn.md

```diff
@@ -17,15 +19,15 @@ Web and Desktop Modeler both offer a similar core BPMN 2.0 Modeling experience:
 - Add BPMN elements from the palette on the left side of the page by dragging and dropping them onto the diagram canvas.
 - Change the type of element in place by clicking on an element to reveal the context menu. Then, click the wrench icon to change the type of element to a [service task](./service-tasks/service-tasks.md) or [user task](./user-tasks/user-tasks.md), for example.
 
-## Demo
+## Creating BPMN elements
 
-![Demo](./assets/demo.gif)
+Using BPMN in Modeler, you can create more BPMN 2.0 elements like task types and event definitions.
```

Comment:
BPMN 2.0 is mentioned on this page, but should we link to it the first time it's referenced? Or is linking to the OMG BPMN page at the top sufficient? I'm on the fence, but this isn't blocking feedback.

Reply:
Linking on first reference at the top of this page should be sufficient -- will adjust.